faf-commons-api/src/main/java/com/faforever/commons/api/dto/NeroxisGeneratorParams.java

```diff
@@ -0,0 +1,16 @@
+package com.faforever.commons.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@EqualsAndHashCode(callSuper = true)
+@Getter
+@Setter
+@JsonTypeName("neroxis")
```

Comment:
Why is this needed? This should be implicit by the JsonSubTypes declaration

Reply:
You are right it is not